gmp/xml.py

```diff
@@ -211,34 +206,28 @@ def create_credential_command(self, name, kwargs):
                 raise ValueError('create_credential requires a '
                                  'private element')
 
-            _xmlKey = etree.SubElement(xmlRoot, 'key')
-            _xmlKeyphrase = etree.SubElement(_xmlKey, 'phrase')
-            _xmlKeyphrase.text = phrase
-            _xmlKeyprivate = etree.SubElement(_xmlKey, 'private')
-            _xmlKeyprivate.text = private
+            _xmlKey = cmd.add_element('key')
+            _xmlKey.add_element('phrase', phrase)
+            _xmlKey.add_element('private', private)
```

Comment:
could you change _xmlKey to user lower case name. This will remove a pylint warning then. In general python doesn't use camelcase besides for class names and in the unittest library.